app/coverage.gradle

```diff
@@ -4,9 +4,15 @@ jacoco {
     toolVersion = "0.7.7.201606060606"
 }
 
+// change these two field if the project uses different build variants for testing
```

Comment:
I think it would be better to state these explicity - "change buildType and productFlavor if the project..."